packages/components/data-table/src/cell.styles.js

```diff
@@ -99,18 +99,21 @@ const getSortableHeaderStyles = (props) => css`
   * GIVEN column is sortable and is not focused
   * THEN AngleUpDown icon is shown (default behaviour)
   * AND AngleUp or AngleDown icon is not shown
-  * 
+  *
   * GIVEN column is sortable and foucsed
   * THEN AngleUpDown icon is hidden
   * AND AngleUp or AngleDown icon is shown
   */
+  svg[id='nonActiveSortingIcon'],
+  svg[id='activeSortingIcon'] {
+    margin-left: ${vars.spacingS};
+    flex-shrink: 0;
```

Comment:
This was the only added this line in these svg styles, to make sure the icon doesn't shrink when the column is narrow.